recipes/hiredis/all/patches/fix-cmake_1.1.0.patch

```diff
@@ -0,0 +1,107 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a8dfaa9..64cd4c4 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -45,23 +45,20 @@ IF(WIN32)
+     ADD_COMPILE_DEFINITIONS(_CRT_SECURE_NO_WARNINGS WIN32_LEAN_AND_MEAN)
+ ENDIF()
+ 
+-ADD_LIBRARY(hiredis SHARED ${hiredis_sources})
+-ADD_LIBRARY(hiredis_static STATIC ${hiredis_sources})
++ADD_LIBRARY(hiredis ${hiredis_sources})
+ 
+ SET_TARGET_PROPERTIES(hiredis
+     PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE
+     VERSION "${HIREDIS_SONAME}")
+-SET_TARGET_PROPERTIES(hiredis_static
```

Comment:
This seems a little overkill, this would also need to be modeled in the package_info to export the correct CMake targets

Reply:
I've submitted https://github.com/redis/hiredis/pull/951 to avoid patches

Reply:
@SpaceIm Thank you for that. I didn't like the patch myself, so your changes are quite welcome. I would actually prefer to not go ahead with this PR at this point until your changes land in `redis/hiredis`.